snap/snapcraft.yaml

```diff
@@ -1,6 +1,6 @@
 name: novnc
 base: core18 # the base snap is the execution environment for this snap
-version: '@VERSION@'
+version: 1.2.0
```

Comment:
We don't want to have to keep updating this, so please keep it as is. It is handled by the deploy job.